### DISCUSSION TITLE
Return all properties in SELECT (aka "SELECT *")?

### DISCUSSION THREAD

**Discussion by divan** [geldata/gel#4904] [Q&A] [answered]
Often when we work with schema, we know it by heart. But many times you either don't remember all the properties or you just learning a new schema you didn't create. In such cases it's really valuable to have a quick way to "discover" schema - i.e. see all object properties, so you can create a shape you need.

In SQL it's just as simple as "SELECT * ..." (often wih /G at the end for tabular view). In EdgeQL I do list `\dt object <ObjectName>` or use EdgeDB UI to explore. But still fill like ability to do something similar to "SELECT *" would be helpful. Autocomplete of properties names would probably help too.

Is it possible to do with current implementation? Maybe there is some special shape for selecting "all first-order properties"?

**Comment by elprans** [comment, marked as answer]
There is an open issue for this: #180
